### ISSUE TITLE
cuda框架下使用adder方法的加速问题

### ISSUE THREAD

**Issue by Lininggggggg** [huawei-noah/AdderNet#28]
请问您有打算发布cuda框架下使用adder方法的代码吗？
有的话请问大概什么时候能发布呢？加速大吗？

**Comment by YunheWang** [comment]
yes, these codes will be released later ;)  may a few weeks or months

The cuda version of addernet can solve both the traning speed (~ x4) and memory usage.

**Comment by Lininggggggg** [comment]
oh, thank you very much!

**Comment by Lininggggggg** [comment]
请问adder方法在测试下能加速吗？现在测试了下也很慢
